pystac/item.py

```diff
@@ -365,9 +365,11 @@ def to_dict(
         d: dict[str, Any] = {
             "type": "Feature",
             "stac_version": pystac.get_stac_version(),
+            "stac_extensions": self.stac_extensions if self.stac_extensions else [],
             "id": self.id,
-            "properties": self.properties,
             "geometry": self.geometry,
+            "bbox": self.bbox if self.bbox is not None else [],
+            "properties": self.properties,
```

Comment:
Need to remove the (now redundant) conditional assignments below: https://github.com/stac-utils/pystac/pull/1423/files#diff-c235f14269d2641c490a6ec16abb7ab96c1f7d4e2ca4514b0ade666b2094469eR377-R381